cpp/oneapi/dal/algo/triangle_counting/backend/cpu/relabel_kernel_cpu.cpp

```diff
@@ -36,12 +36,11 @@ template void parallel_prefix_sum<__CPU_TAG__>(const std::int32_t* degrees_relab
                                                std::int64_t num_blocks,
                                                std::int64_t vertex_count);
 
-template void fill_relabeled_topology<__CPU_TAG__>(const std::int32_t* vertex_neighbors,
-                                                   const std::int64_t* edge_offsets,
-                                                   std::int32_t* vertex_neighbors_relabel,
-                                                   std::int64_t* edge_offsets_relabel,
-                                                   std::int64_t* offsets,
-                                                   const std::int32_t* new_ids,
-                                                   std::int64_t vertex_count);
+template void fill_relabeled_topology<__CPU_TAG__>(
+    const dal::preview::detail::topology<std::int32_t>& t,
+    std::int32_t* vertex_neighbors_relabel,
+    std::int64_t* edge_offsets_relabel,
```

Comment:
I assume 2-4 function arguments are used to return the result. If so, are there any guidelines that indicate how to group function arguments by in/in-out/out attribute?

Reply:
Oneapi coding style does not cover it. Suggestion - to follow std ideology: input is always first, output is second. Applied both for template and functional arguments. Example:
template< class InputIt, class OutputIt >
constexpr OutputIt copy( InputIt first, InputIt last, OutputIt d_first );

That is - in, in-out, out would be a conformant order